src/test/Components/CreateImageWizard/steps/Packages/Packages.test.tsx

```diff
@@ -329,33 +347,33 @@ describe('Step Packages', () => {
     await screen.findByText('The search value is too short');
   });
 
-  test('should display relevant results in selected first', async () => {
```

Comment:
And a proof that this also still works in browser

![image](https://github.com/user-attachments/assets/4da3aa69-d7c3-474a-8f8a-3cca2a743d83)